library/stringUtils.sol

```diff
@@ -24,4 +24,32 @@ library StringUtils {
     function equal(string _a, string _b) returns (bool) {
         return compare(_a, _b) == 0;
     }
+    /// @dev Finds the first occurrence of _b in _a and returns its index or -1 if failure
+    function indexOf(string _a, string _b) returns (int) // _a = string to search, _b = string we want to find
```

Comment:
Some libraries use the names "_haystack" and "_needle" for those variables, which makes the comment mostly unnecessary.